openhtf/exe/test_state.py

```diff
@@ -50,24 +48,24 @@ class TestState(object):
     config: The config being used for this test.
     test: openhtf.Test instance describing the test to run.
   """
-  State = Enum(
+  Status = Enum(
```

Comment:
There's two enums called Status, it seems one is more a TestExecutionStatus and one is TestResultStatus (or without Test prefix)?